operator/redisfailover/service/names.go

```diff
@@ -8,11 +8,17 @@ import (
 
 // GetRedisName returns the name for redis resources
 func GetRedisName(rf *redisfailoverv1alpha2.RedisFailover) string {
+	if rf.Spec.Redis.ConfigMap != "" {
```

Comment:
This function is widely used and cannot be changed if the ConfigMap is wanted by the user.

Doing this, all Redis resources would be called as the ConfigMap.